extension/scripts/features/highlight-blood-bags/ttHighlightBloodBags.js

```diff
@@ -87,10 +88,10 @@
 	}
 
 	async function removeHighlights() {
-		for (const highlight of document.findAll(".good-blood, .bad-blood")) {
+		for (const highlight of document.findAll(".items-wrap .good-blood, .items-wrap .bad-blood, #armoury-medical .good-blood, #armoury-medical .bad-blood")) {

```

Comment:
Why make it more specific? Those classes are only used for this feature anyway.

Reply:
It removes the classes for Quick Items items too.